src/es/loadDataset.ts

```diff
@@ -0,0 +1,5 @@
+import stringToStream from 'string-to-stream'
+
+export async function loadDatasetStream (prefix) {
+  return stringToStream((await import(`../../ontologies/${prefix}.nq`)).default)
```

Comment:
Dynamic import requires configuration on the consumer end. Will add a note about that in the readme

Reply:
Is this solved?

Reply:
Yes

https://github.com/zazuko/rdf-vocabularies/pull/8/files#diff-04c6e90faac2675aa89e2176d2eec7d8R21